files/web/static/js/cr/main/administrado/resursoj.js

```diff
@@ -6,7 +6,15 @@ $(function () {
         url: '/api/resursoj/list',
         select: [ 'id', 'name', 'description', 'url' ],
         defaultOrder: [ 1, 'asc' ],
+        dataFormatter: function (val, col) {
```

Comment:
Ĉar mi ne trovis rektan manieron por facile enmeti "pre" stilo en la tria kolumno, mi ŝanĝis ĉi tie kiel la teksto aperas por solvi https://github.com/tejoesperanto/centra-reto/pull/5#discussion_r276935016.